src/components/rightSidebar/Users.vue

```diff
@@ -0,0 +1,79 @@
+<template xmlns:v-slot="http://www.w3.org/1999/XSL/Transform">
+  <div>
+    <v-text-field
+      label="Search"
+      single-line
+      v-model="userNameLike"
+      :value="userNameLike"
+    ></v-text-field>
```

Comment:
you can write <foo /> instead of <foo></foo>. Also for `v-progress-circular`.

Reply:
Didn't quite understand this issue. Can you explain, please?

Reply:
I edited initial comment. Please check it once again. It was bad formatted.

Reply:
Fixed.
(WebStorm advised this, because `Empty tag doesn't work in some browsers`)

Reply:
Yes, but this rule is not applicable to frameworks. You can disable "Empty tag doesn't work in some browsers" warning in config.